packages/cyberstorm/src/components/AntiResult/AntiResult.tsx

```diff
@@ -0,0 +1,26 @@
+import { ReactNode } from "react";
+import styles from "./AntiResult.module.css";
+import { classnames } from "../../utils/utils";
+
+export interface Props {
+  children?: ReactNode | ReactNode[];
+  className?: string;
+}
+
+/**
+ * Ease of use component set for adding helpful content in
```

Comment:
Not so sure they're that helpful, at least in the first usecase 😅

Reply:
Well alright maybe the ease of use is a bit of a lie, but still the component set will provide a centralized place to manipulate all of the places where results are none.

To be iterated on 👌

Reply:
I am 99% certain that no one will iterate on that.